app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,23 @@
+class MoviesController < ApplicationController
+  def index
+    @category   = params[:category] || 'title'
+    @sort_order = params[:sort_order] || 'asc'
+    @search     = params[:search]
+    @active     = 'movies'
```

Comment:
If you're not using these variables in your view, make them local variables. Looking at this, I would assume you're calling them in the view.